ERCS/erc-8023.md

```diff
@@ -0,0 +1,138 @@
+	/// stored as pendingOwner.
+	function confirmOwnershipTransfer(address newOwner) external;
+	
+	/// @dev cancels the pending ownership transfer. Before the final step of ownership transfer (acceptOwnershipTransfer()).
+	/// This function should wipe out the pendingOwner.
+	/// By calling this function, ownership transfer process is canceled and should be reinitiated from initiateOwnershipTransfer().
+	function cancelPendingOwnershipTransfer() external;
+
+	/// @dev accepts the ownership transfer. Final step of ownership transfer.
+	/// This function can only be called by the newOwner that was confirmed in step 2.
+	/// The contract should perform access control e.g.,
+	/// msg.sender == pendingOwner()
+	function acceptOwnershipTransfer() external;
+	
+	/// @notice only the address returned by owner() has authority as the owner.
+	/// pendingOwner() and preConfirmedOwner() should not possess any
+	/// authority/access/right.
+	/// @dev returns the owner of the contract
+	function owner() external view returns (address);
+	
+	/// @dev returns the pending owner of the account.
+	/// pending owner should not have any authority/access/right.
+	function pendingOwner() external view returns (address);
+	
+	/// @dev returns the pre-confirmed owner of the account.
+	/// pre-confirmed owner should not have any authority/access/right.
+	function preConfirmedOwner() external view returns (address);
+	
+	/// @dev returns the ownership transfer buffer time (in seconds).
+	/// the buffer is enforced between initiation <> confirmation of ownership transfer. 
+	/// the standard does not enforce the value range. it is highly recommended to be between 2 <> 14 days.
+	function getOwnershipTransferBuffer() external view returns (uint256);
+}
+```
+
+The `MultiStepOwnable` contract MAY implement the `UpdateableOwnershipTransferBuffer` interface to enable buffer period modification.
+
+The contract MUST update the buffer period with initiation and then confirmation after the existing buffer period.
+If this behavior is not enforced, the security of `ownershipTransferBuffer` could break during owner key compromise scenario.
+
+```solidity
+/// @title UpdateableOwnershipTransferBuffer. Extension of MultiStepOwnable.
+contract UpdateableOwnershipTransferBuffer {
+
+	/// @dev initiates the update of ownership transfer time buffer.
+	function initiateOwnershipBufferUpdate(uint256 newBuffer) external;
+	
+	/// @dev confirms the update of ownership transfer time buffer.
+	/// confirmation SHOULD revert if existing time buffer did not pass since
+	/// initiation of ownership transfer time buffer.
+	function confirmOwnershipBufferUpdate(uint256 newBuffer) external;
+}
+```
+
+## Rationale
+
+We define the main objective of this standard to make ownership more secure and handled in a multi-stepped approach that enables the operation to be conducted with more caution and lesser operational mistakes, while being immune to potential scam attacks.
```

Comment:
The "main objective" of a proposal should live in the Motivation section. The Rationale section should be used to explain choices made within the proposal itself. My pet analogy is:

> Motivation: We need to build a shed because...
> Rationale: We chose to paint the shed red because...